pkg/machines/actions.es6

```diff
@@ -83,9 +91,16 @@ export function vmDesktopConsole(vm, consoleDetail) {
  * and scheduled on later.
  *
  * @param action I.e. getAllVms()
+ * @param timeout Non-default timeout
+ * @param name VM name (if nonexisting or usage not enabled, action will not be run);
+ *             if not given, use global timeout
+ * @param connectionName Connection for #name
  */
-export function delayPolling(action, timeout) {
+export function delayPolling(action, timeout, name, connectionName) {
     return (dispatch, getState) => {
+        if (name && !usagePollingEnabled(getState(), name, connectionName))
```

Comment:
I think it would be better to have this test in `doUsagePolling` and not here since `delayPolling` is a general purpose utility.  (Alternatively, the whole test for when to stop could be handed in as a callback, but that would just be obscure, I think.)